docs/asciidoc/modules/ROOT/pages/cypher-execution/cypher-based-procedures-functions.adoc

```diff
@@ -27,7 +27,7 @@ you have to quote the result, e.g `CALL apoc.custom.declareProcedure("procWithNu
 
 .Type Names
 The `typeParam` and `typeResult` in the signature parameter can be one of the following values:
-* FLOAT, DOUBLE, INT, INTEGER, NUMBER, LONG
+* FLOAT, DOUBLE, INT, INTEGER, INTEGER | FLOAT, LONG
```

Comment:
Please add a comment were we say that we still use number for backwards compatibility